packages/features/calendars/CalendarSwitch.tsx

```diff
@@ -60,6 +66,7 @@ const CalendarSwitch = (props: ICalendarSwitchProps) => {
         await utils.viewer.integrations.invalidate();
```

Comment:
i think you would also have to add await utils.viewer.connectedCalendars.invalidate(); here

Reply:
Good spot - testing

Reply:
Commited